### ISSUE TITLE
Couldn't find any logs waiting 60 secs before checking again Unhandled Exception Occurred.

### ISSUE THREAD

**Issue by Pujith01** [mohankumarpaluru/oracle-freetier-instance-creation#39]
I did all the configuration in VM.Standard.E2.1.Micro. It installed all the dependencies and in the end, it gave this unhandled exception.

**Comment by m4g3s** [comment]
Remove # TODO from your oci_config file

`key_file=/home/ubuntu/oracle-freetier-instance-creation/oci_api_private_key.pem `

This solved the issue for me if it doesn't for you it may be another issue you can do

`source .venv/bin/activate
python3 main.py`

And you should see an output error that will tell you what is wrong.

**Comment by mohankumarpaluru** [comment]
@Pujith01 if its 401 , get a new API key and try again

**Comment by 1watobi** [comment]
I am having the same issue. This is the error

**Comment by Yorksik** [comment]
<img width="1039" alt="image" src="https://github.com/user-attachments/assets/edebe781-4a65-49a5-802a-977636bfba3e">

I'm getting this error.

**Comment by heeyoung-choi** [comment]
![image](https://github.com/user-attachments/assets/c1543637-64d2-4cc1-a911-f384b85d61e7)
this is my error

**Comment by Pxnjo** [comment]
> <img alt="image" width="1039" src="https://private-user-images.githubusercontent.com/17272516/392985753-edebe781-4a65-49a5-802a-977636bfba3e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc3NjEwMTgsIm5iZiI6MTczNzc2MDcxOCwicGF0aCI6Ii8xNzI3MjUxNi8zOTI5ODU3NTMtZWRlYmU3ODEtNGE2NS00OWE1LTgwMmEtOTc3NjM2YmZiYTNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI0VDIzMTgzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVkNTU5ZTRhNTM4OGZjNjU2NWEzYTU4MDI4NGE4NDc2YmExYWI4MTY1OWQ4NWFkZmE5NTYxY2RjMjFlOWU3NWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8NuAvnfCsTAp3bujwoViPjS2N5lR56RkGAvY58PJ7Go">
> I'm getting this error.

i solved using ' 
OPERATING_SYSTEM='Canonical Ubuntu'

it seems work

**Comment by AzpektDev** [comment]
> > <img alt="image" width="1039" src="https://private-user-images.githubusercontent.com/17272516/392985753-edebe781-4a65-49a5-802a-977636bfba3e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc3NjEwMTgsIm5iZiI6MTczNzc2MDcxOCwicGF0aCI6Ii8xNzI3MjUxNi8zOTI5ODU3NTMtZWRlYmU3ODEtNGE2NS00OWE1LTgwMmEtOTc3NjM2YmZiYTNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI0VDIzMTgzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVkNTU5ZTRhNTM4OGZjNjU2NWEzYTU4MDI4NGE4NDc2YmExYWI4MTY1OWQ4NWFkZmE5NTYxY2RjMjFlOWU3NWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8NuAvnfCsTAp3bujwoViPjS2N5lR56RkGAvY58PJ7Go">
> > I'm getting this error.
> 
> i solved using ' OPERATING_SYSTEM='Canonical Ubuntu'
> 
> it seems work

Yes, I think having the name without the `'` is  causing the crash. After adding it in oci.env it seems to work

**Comment by btarg** [comment]
> Remove # TODO from your oci_config file
> 
> `key_file=/home/ubuntu/oracle-freetier-instance-creation/oci_api_private_key.pem `
> 
> This solved the issue for me if it doesn't for you it may be another issue.
> 
> do the following in console
> 
> ```
> source .venv/bin/activate
> 
> python3 main.py
> ```
> 
> And you should see an output error that will tell you what is wrong.

Thank you for this!
My issue was that I had double-quotes in my key's path.
the shell script really ought to actually give you the error traceback!

**Comment by mohankumarpaluru** [comment]
Addressed in PR #90 (2a4a5d4) with a new troubleshooting guide.
